.editorconfig

```diff
@@ -13,3 +13,12 @@ tab_width = 4
 # Markup files
 [{*.html,*.xml,*.xml.in,*.yml}]
 tab_width = 2
+
+# Ignore test assets (These files need to remain unformatted)
+[/tests/assets/*]
+charset = unset
+end_of_line = unset
+indent_size = unset
+indent_style = unset
+insert_final_newline = unset
+tab_width = unset
```

Comment:
Uhm EditorConfig does not apply a real formatting, it only gives hints to the code editors about the preferences to use in each file… and unsetting everything will only make that whoever edits the file will contaminate it with the preferences of their editor.

Reply:
I pinpointed the issue to being the insertion of a new line so I'll explicitly set the final new line rule to false instead.